web-console/src/app/(root)/(authenticated)/(app)/connectors/create/page.tsx

```diff
@@ -62,6 +62,12 @@ const ConnectorCreateGrid = () => {
             addOutput={{ href: '#output/snowflake' }}
             data-testid='box-connector-snowflake'
           />
+          {/* <AddConnectorCard
```

Comment:
This block is added and commented out intentionally, will be uncommented in the next PR